docker/example/supervisord/Dockerfile

```diff
@@ -10,14 +10,14 @@ LABEL version="${version}"
 ENV DEBIAN_FRONTEND=noninteractive \
     TZ=America/New_York \
     HOME=/root
-RUN (echo $TZ > /etc/timezone) \
- && dpkg-reconfigure tzdata
+RUN (echo $TZ > /etc/timezone)
 
 # Install packages
 RUN apt-get update \
  && apt-get -y upgrade \
  && apt-get -y install \
 	supervisor \
+  tzdata \
```

Comment:
Sorry to keep nitpicking, but you could you fix the indentation to line up with "supervisord" on the previous line? Thank you!